lib/configurators/common.go

```diff
@@ -47,6 +48,10 @@ type BootstrapFlags struct {
 	ForceMemoryDBPermissions bool
 	// ForceEC2Permissions forces the presence of EC2 permissions.
 	ForceEC2Permissions bool
+	// ForceAWSKeyspacesPermissions forces the presence of AWS Keyspaces permissions.
+	ForceAWSKeyspacesPermissions bool
+	// ForceDynamoDBPermissions forces the presence of DynamoDB permissions.
+	ForceDynamoDBPermissions bool
```

Comment:
Are the new booleans used here?

If not, could we delay them to a PR where they would be used?

Reply:
mistake when splitting the commits on my part. I'll push the changes that use these bools

Reply:
fixed in 686a5f0a18b9003ac0fea791f0f172870721fdab